docs/getting_started/index.md

```diff
@@ -168,7 +168,8 @@ Here's a simple GUI set up for our "Hello World" scenario:
 
     === "Python"
 
-        Taipy pages can be defined in multiple ways: Markdown, Html or Python. *page* is built through the Taipy Gui Builder.
+        Taipy pages can be defined in multiple ways: Markdown, Html or Python. *page* is built through 
```

Comment:
I don't think this line is useful, and the code does not define the tgb symbol.
If you really need to reference the taipy.gui.builder module, the proper wording is 'Page Builder API'. **Not GUI Builder**